app/models/user.rb

```diff
@@ -0,0 +1,28 @@
+require 'dm-validations'
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  property :id,                  Serial
+  property :username,            String, required: true, unique: true
+  property :name,                String, required: true
+  property :email,               String, required: true, format: :email_address, unique: true
```

Comment:
Metrics/LineLength: Line is too long. [93/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)